clang/test/CodeGenOpenCL/builtins-alloca.cl

```diff
@@ -0,0 +1,255 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
+// RUN: %clang_cc1 %s -O0 -triple amdgcn-amd-amdhsa -cl-std=CL1.2 \
+// RUN:     -emit-llvm -o - | FileCheck --check-prefix=OPENCL12 %s
+// RUN: %clang_cc1 %s -O0 -triple amdgcn-amd-amdhsa -cl-std=CL2.0 \
+// RUN:     -emit-llvm -o - | FileCheck --check-prefix=OPENCL20 %s
+// RUN: %clang_cc1 %s -O0 -triple amdgcn-amd-amdhsa -cl-std=CL3.0 \
+// RUN:     -emit-llvm -o - | FileCheck --check-prefix=OPENCL30 %s
```

Comment:
Should add a comment prefix. I don't think you should get different code in any case

Reply:
Can you let me know what really need to be tested via this test case, so exactly I could trim out the unnecessary portion of comment prefix.

Reply:
I meant common prefix. -check-prefixes=OPENCL,OPENCL12 etc. But I don't think you should ever get different code

Reply:
yeah, that's true, as now no more address space casting is happening for any version of OpenCL!